data/page_text.yaml

```diff
@@ -18,72 +18,99 @@ gdp_processing_before: >-
   The GDP moves computationally intensive processing steps from a researcher's computer to
   the web. This means researchers have more computational power at their fingertips, and 
   do not need to need their computer tied up during a lengthy processing step.
+  
 gdp_processing_after: >-
   
   This differs from traditional geoprocessing workflows which require downloads of an 
   immense amount of data to then process. The GDP requires users to download only the 
   processed data.
```

Comment:
..."which can require download of massive amounts of data to be subset and processed."